Application/src/Utils/Text.js

```diff
@@ -84,3 +32,41 @@ export async function getText(image) {
 
   return obj;
```

Comment:
Can you please rename the variable from "obj" that tells me nothing that the variable to something that describes the data being returned. Also can you rename cg to colorGrade as that was a perfect name for the attribute. The "images" variable is also a vague name for a variable, can you please be more descriptive with your variable names? I'd rather have long descriptive names, rather than short concise ones any day of the week.